chapters/002-smol-internet/README.md

```diff
@@ -367,25 +309,16 @@ We can see that the ping from `boudi` made it to `tara`! Huzzah!
 
 ### Can `tara` ping `boudi`?
 
-Now... let's see if `tara` is able to ping `boudi`. We're going to do the same 3
-window setup, but this time `tara` will be running both `ping` and `tcpdump`.
+Now... let's see if `tara` is able to ping `boudi`. We're going to do the same 3 window setup, but this time `tara` will be running both `ping` and `tcpdump`.
 
-The interesting thing with `tara` pinging `boudi` is that `boudi` has 2 network
-IP addresses we can ping. Which one should we use? Well, 10.1.1.3 is the IP
-that's on the `squasheeba` network. `tara`, unfortunately, doesn't know anything
-about how to reach this network, which means, if we try to ping it, `boudi`
-won't receive the ping and `tara` is just screaming into the void. If instead,
-we use `boudi`'s IP on `doggonet`, 10.1.2.3, we should have a successful result.
+The interesting thing with `tara` pinging `boudi` is that `boudi` has 2 network IP addresses we can ping. Which one should we use? Well, 10.1.1.3 is the IP that's on the `squasheeba` network. `tara`, unfortunately, doesn't know anything about how to reach this network, which means, if we try to ping it, `boudi` won't receive the ping and `tara` is just screaming into the void. If instead, we use `boudi`'s IP on `doggonet`, 10.1.2.3, we should have a successful result.
 
 1. `hopon tara` will run `ping 10.1.2.3` 
 2. `hopon tara` will run `tcpdump -ne`
 3. `hopon boudi` will run `tcpdump -ni eth0`
 
 **NOTICE** 
-We added the `-e` flag to our `tcpdump` command for `tara`. Why? That flag
-reveals information about ethernet headers in each packet. If we look at the
-network interface information for `boudi`, we can see the mac address for its
-connection on `doggonet`: 02:42:0a:01:02:03.
+We added the `-e` flag to our `tcpdump` command for `tara`. Why? That flag reveals information about ethernet headers in each packet. If we look at the network interface information for `boudi`, we can see the mac address for its connection on `doggonet`: 02:42:0a:01:02:03.
```

Comment:
1. should we link to the ip-and-mac-addresses.md file in our appendix here with a bit of a callback to ARP?
2. should we move our "how to read mac addresses on docker" explanation we did in chapter 4 to here?